releasenotes/notes/0.17/prepare-0.17-2ab9429b69e1d25c.yaml

```diff
@@ -0,0 +1,28 @@
+---
+prelude: |
+    The Qiskit Terra 0.17.0 includes many new features and bug fixes. The major
+    new feature for this release is the introduction of the
+    :mod:`qiskit.algorithms` and :mod:`qiskit.opflow` modules which were
+    migrated and adapted from the :mod:`qiskit.aqua` project.
+features:
+  - |
+    A new module :mod:`qiskit.algorithms` has been introduced. This module
+    contains functionality equivalent to what has previously been
+    provided by the :mod:`qiskit.aqua.algorithms` module (which is now
+    deprecated) and provides the building blocks for constructing quantum
+    algorithms. For details on migrating from ``qiskit-aqua`` to this new
+    module please refer to the migration guide "INSERT LINK HERE"
+  - |
+    A new module :mod:`qiskit.opflow` has been introduced. This module
+    contains functionality equivalent to what has previously been
+    provided by the :mod:`qiskit.aqua.operators` module (which is now
+    deprecated) and provides the operators and state functions which are
+    used to build quantum algorithms. For details on migrating from
+    ``qiskit-aqua`` to this new module please refer to the migration guide
+    "INSERT LINK HERE"
```

Comment:
we should fill this in

Reply:
This is intentional and unfortunate side effect of our split documentation builds, see my comments starting here: https://github.com/Qiskit/qiskit-terra/pull/6075#discussion_r605246560

I will update this in the release notes post release.